src/views/ChecksumTab.vue

```diff
@@ -28,10 +28,11 @@
 			:options="algorithms"
 			track-by="id"
 			label="label"
-			@input="onAlgorithmChangeHandler" />
+			@update:modelValue="onAlgorithmChangeHandler" />
 		<br>
 		<br>
-		<p :class="{ 'icon-loading': loading }" class="checksum-hash-result" @click="clipboard">
+		<NcLoadingIcon v-if="loading" :size="40" />
+		<p v-else class="checksum-hash-result" @click="clipboard">
```

Comment:
This change is not necessarily required for Vue 3 but the loading classes have been deprecated and should not be used anymore.